Representor/HTTP/Adapters/HTTPSirenAdapter.swift

```diff
@@ -0,0 +1,161 @@
+//
+//  SirenAdapter.swift
```

Comment:
Right, but it is HTTP folder... anyway do you plan to rename - move Hal adapter too?